invenio_iiif/__init__.py

```diff
@@ -6,7 +6,91 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""IIIF API for Invenio."""
+"""Image cropping and previewer API.
+
+Invenio-IIIF integrates
+`Invenio-Records-Files <https://invenio-records-files.readthedocs.io>`_ with
+`Flask-IIIF <https://flask-iiif.readthedocs.io/en/latest/>`_
+to provide an endpoint for serving images
+with the International Image Interoperability Framework (IIIF) API standards.
+To be a bit more specific Invenio-IIIF is registering the provided endpoints
+from Flask-IIIF and it is using the files permissions provided
+by Invenio-Files-Rest.
+On top of that it is providing additional functionalities based on the IIIF
+standards. For more info check here (https://iiif.io/)
+
+Initialization
+--------------
+When this package is installed with Flask-IIIF it will register the second's
+blueprints under the configurable prefix in config['IIIF_API_PREFIX']
+
+Usage
+-----
+*Prerequisites*: Having an image or pdf stored under an Invenio File Object
```

Comment:
We could point to documentation how to create Invenio File Object or just add a few lines with instructions here.